mii/method_table.py

```diff
@@ -54,8 +54,11 @@ def pack_request_to_proto(self, request_dict, **query_kwargs):
     def unpack_request_from_proto(self, request):
         return request
 
-    def run_inference(self, inference_pipeline, args, kwargs):
-        return inference_pipeline(*args, **kwargs)
+    def run_inference(self, inference_pipeline, args, kwargs, is_non_persistent=False):
```

Comment:
We should be able to avoid needing this extra `is_non_persistent` if we modify the `args` value in `MIINonPersistentClient`

Reply:
I am currently modifying the args value like this in the MIINonPersistentClient class. 
```python 
else:
            query = request_dict['query']
            kwargs = {}
            return task_methods.run_inference(inference_pipeline, query, kwargs, True)
```
But even with this, there is an error when I try to pass query in as *args to the run_inference call in method_table (for the non_persistent models).